azure/packages/azure-service-utils/src/test/types/validateAzureServiceUtilsPrevious.generated.ts

```diff
@@ -41,62 +41,6 @@ type TypeOnly<T> = T extends number
 					[P in keyof T as SkipUniqueSymbols<P>]: TypeOnly<T[P]>;
 				};
 
-/*
- * Validate forward compatibility by using the old type in place of the current type.
- * If this test starts failing, it indicates a change that is not forward compatible.
- * To acknowledge the breaking change, add the following to package.json under
- * typeValidation.broken:
- * "InterfaceDeclaration_IUser": {"forwardCompat": false}
- */
-declare function get_old_InterfaceDeclaration_IUser():
-    TypeOnly<old.IUser>;
-declare function use_current_InterfaceDeclaration_IUser(
-    use: TypeOnly<current.IUser>): void;
-use_current_InterfaceDeclaration_IUser(
-    get_old_InterfaceDeclaration_IUser());
-
-/*
- * Validate backward compatibility by using the current type in place of the old type.
- * If this test starts failing, it indicates a change that is not backward compatible.
- * To acknowledge the breaking change, add the following to package.json under
- * typeValidation.broken:
- * "InterfaceDeclaration_IUser": {"backCompat": false}
- */
-declare function get_current_InterfaceDeclaration_IUser():
-    TypeOnly<current.IUser>;
-declare function use_old_InterfaceDeclaration_IUser(
-    use: TypeOnly<old.IUser>): void;
-use_old_InterfaceDeclaration_IUser(
-    get_current_InterfaceDeclaration_IUser());
-
-/*
- * Validate forward compatibility by using the old type in place of the current type.
- * If this test starts failing, it indicates a change that is not forward compatible.
- * To acknowledge the breaking change, add the following to package.json under
- * typeValidation.broken:
- * "EnumDeclaration_ScopeType": {"forwardCompat": false}
- */
-declare function get_old_EnumDeclaration_ScopeType():
-    TypeOnly<old.ScopeType>;
-declare function use_current_EnumDeclaration_ScopeType(
-    use: TypeOnly<current.ScopeType>): void;
-use_current_EnumDeclaration_ScopeType(
-    get_old_EnumDeclaration_ScopeType());
-
-/*
- * Validate backward compatibility by using the current type in place of the old type.
- * If this test starts failing, it indicates a change that is not backward compatible.
- * To acknowledge the breaking change, add the following to package.json under
- * typeValidation.broken:
- * "EnumDeclaration_ScopeType": {"backCompat": false}
- */
-declare function get_current_EnumDeclaration_ScopeType():
-    TypeOnly<current.ScopeType>;
-declare function use_old_EnumDeclaration_ScopeType(
-    use: TypeOnly<old.ScopeType>): void;
-use_old_EnumDeclaration_ScopeType(
-    get_current_EnumDeclaration_ScopeType());
-
```

Comment:
This might be instability in 'fluid-type-test-generator'.  I'm waiting to see what CI says.

Reply:
This might be relevant: https://github.com/microsoft/FluidFramework/pull/20990#discussion_r1591318714 Also, not sure if you want to wait for that PR to merge before this one. I don't think it's necessary.